bib/cmd/bootc-image-builder/image.go

```diff
@@ -168,6 +168,8 @@ func pipelinesForISO(c *ManifestConfig, rng *rand.Rand) (image.ImageKind, error)
 	}
 
 	img := image.NewAnacondaContainerInstaller(containerSource, ref)
+	img.SquashfsCompression = "zstd"
```

Comment:
This commit sets the compression to `zstd` gut also adds the `lz4` package to the ISO, which is a bit weird (leftover from trying out `lz4` I imagine).

Reply:
Good catch, removed the `lz4` package from that commit.